iphone/Classes/TiAppiOSProxy.m

```diff
@@ -544,6 +551,39 @@ -(void)didRegisterUserNotificationSettingsNotification:(NSNotification*)notifica
          withObject:[self formatUserNotificationSettings:(UIUserNotificationSettings*)[notificationSettings object]]];
 }
 
+#pragma mark Apple Watchkit notifications
+
+-(void)didReceiveWatchExtensionRequestNotification:(NSNotification*)notif
+{
+    [self fireEvent:@"watchkitextensionrequest" withObject:[notif userInfo]];
+}
+
+#pragma mark Apple Watchkit handleWatchKitExtensionRequest reply
+
+-(void)sendWatchExtensionReply:(id)args
+{
+    if(![TiUtils isIOS8OrGreater]) {
+        return;
+    }
+    
+    enum Args {
+        kArgKey = 0,
+        kArgCount,
+        kArgUserInfo = kArgCount
+    };
+    
+    ENSURE_TYPE(args,NSArray);
+    ENSURE_ARG_COUNT(args, kArgCount);
+    
+    NSString *key = [TiUtils stringValue:[args objectAtIndex:kArgKey]];
+    NSDictionary *userInfo = [args objectAtIndex:kArgUserInfo];
```

Comment:
The `kArgUserInfo` value is 1 so it should be inside the `if([args count] > 1)` block